.vscode/settings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "cmake.configureEnvironment": {
```

Comment:
ah, Feel good to see this kind of vscode PRs!

I tried to add launch.json and tasks.json in the past so customers could already have debugging env ready to launch debugger with vscode.

But the team didn't see it as a good idea to add this kind of utilities.

As a customer, I like when I am given all of this things that everyone needs to eventually figure it out

Reply:
Yea the only nit I have with the build.json and tasks.json is that they are VERY involved and in some ways require a different lesson in how to build the SDK. Doing it this way, the user just sets various options and the extension handles it for them. I think it's overall an easier system to maintain and easier for the user to use.